js/components/thumbnail.js

```diff
@@ -7,12 +7,107 @@ var React = require('react'),
     Utils = require('./utils');
 
 var Thumbnail = React.createClass({
+  getInitialState: function() {
+    this.positionY = 0;
```

Comment:
Where are these numbers coming from?

Reply:
It is default value for ideal situation.